staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go

```diff
@@ -150,6 +150,16 @@ var (
 		[]string{"requestKind"},
 	)
 
+	// WindowStats from the last window
+	concurrencyGauges = compbasemetrics.NewGaugeVec(
+		&compbasemetrics.GaugeOpts{
+			Name:           "apiserver_current_inflight_request_measures",
+			Help:           "Stats on number of requests executing by mutating or not, measure.",
+			StabilityLevel: compbasemetrics.ALPHA,
+		},
+		[]string{"requestKind", "measure"},
+	)
```

Comment:
I find this a bit confusing as an instrumentation pattern. I would prefer different metrics for each 'measure'. These are really heterogenous dimensions that you are measuring, so throwing them into the same metric is really quite confusing (it will also not visualize well).

Reply:
OK, I did that in the force-push to https://github.com/kubernetes/kubernetes/commit/cdea2166bd19aa4311e3756d52a69a05ae1a80ed

Reply:
it looks like this metric still includes measure.

Reply:
Oh, right, I only updated the other.  Will do this too.

Reply:
done, and more